sdk/python/feast/infra/online_stores/contrib/mysql_online_store/mysql.py

```diff
@@ -82,21 +75,95 @@ def unpack_write_entity(self, entity: EntityKeyProto,
             created_ts = _to_naive_utc(created_ts)
         return entity_key_bin, values, ts, created_ts
 
+    def online_write_batch_cooperative(
+        self,
+        config: RepoConfig,
+        table: FeatureView,
+        data: List[
+            Tuple[EntityKeyProto, Dict[str, ValueProto], datetime, Optional[datetime]]
+        ],
+        old_data:  Optional[List[
+            Tuple[EntityKeyProto, Dict[str, ValueProto], datetime, Optional[datetime]]
+        ]]
+    ) -> bool:
+        if old_data and len(data) != len(old_data):
+            raise ValueError(f'More columns in new entry {len(data)} than old entry {len(old_data)}')
+
+        conn = self._get_conn(config)
+        cur = conn.cursor()
+
+        project = config.project
+        for i in range(len(data)):
```

Comment:
@Rostifar did you add in these changes? It looks like it from the commit history but I just want to make sure because i just now noticed it.